keyboards/1upkeyboards/1up60hte/config.h

```diff
@@ -21,7 +21,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x6F75 // OU
 #define PRODUCT_ID      0x6060
```

Comment:
```suggestion
#define PRODUCT_ID      0x6874 // HT
```

?

Reply:
why are we changing the product id?

Reply:
I figured the HSE got a selected one, so why not do the same for this one?

Reply:
HSE got one cuz its product id was like all 0000 if I remember correctly, but this one is directly from the 1upkeyboard kbfi****** json file.

Reply:
KBF JSON files don't include preset USB Device ID data.

If the 1up60RGB gets VIA support added, its Product ID will have to be changed (it's currently identical to the HTE's).

Reply:
alright good enough reason for me.